vulnerabilities/tests/test_openssl.py

```diff
@@ -0,0 +1,220 @@
+                                                  qualifiers=OrderedDict(), subpath=None),
+                                       PackageURL(type='openssl', namespace=None, name='openssl',
+                                                  version='1.0.2q',
+                                                  qualifiers=OrderedDict(), subpath=None),
+                                       PackageURL(type='openssl', namespace=None, name='openssl',
+                                                  version='1.1.1b',
+                                                  qualifiers=OrderedDict(), subpath=None),
+                                       PackageURL(type='openssl', namespace=None, name='openssl',
+                                                  version='1.0.2h',
+                                                  qualifiers=OrderedDict(), subpath=None),
+                                       PackageURL(type='openssl', namespace=None, name='openssl',
+                                                  version='1.0.2o',
+                                                  qualifiers=OrderedDict(), subpath=None),
+                                       PackageURL(type='openssl', namespace=None, name='openssl',
+                                                  version='1.0.2e',
+                                                  qualifiers=OrderedDict(), subpath=None),
+                                       PackageURL(type='openssl', namespace=None, name='openssl',
+                                                  version='1.0.2d',
+                                                  qualifiers=OrderedDict(), subpath=None),
+                                       PackageURL(type='openssl', namespace=None, name='openssl',
+                                                  version='1.0.2n',
+                                                  qualifiers=OrderedDict(), subpath=None)},
+                resolved_package_urls={
+                    PackageURL(type='openssl', namespace=None, name='openssl', version='1.1.1e',
+                               qualifiers=OrderedDict(), subpath=None),
+                    PackageURL(type='openssl', namespace=None, name='openssl', version='1.0.2u',
+                               qualifiers=OrderedDict(), subpath=None)},
+                reference_urls=['https://github.com/openssl/openssl/commit/' +
+                                '419102400a2811582a7a3d4a4e317d72e5ce0a8f',
+                                'https://github.com/openssl/openssl/commit/' +
+                                'f1c5eea8a817075d31e43f5876993c6710238c98'],
+                reference_ids=[],
+                cve_id='CVE-2019-1551')
+        ]
+        found_data = OpenSSLDataSource.to_advisories(data)
+
+        # Sort them by CVE-ID
+        found_data.sort(key=lambda x: x.cve_id)
+        expected_data.sort(key=lambda x: x.cve_id)
+
+        # Check first advisory
+        assert found_data[0].cve_id == expected_data[0].cve_id
```

Comment:
question: Won't these be same as doing a single `assert found_data == expected_data` ? If yes do use  `assert found_data == expected_data`